tests/unit/resources/storage/test_storage_volume_templates.py

```diff
@@ -102,7 +102,7 @@ def test_get_connectable_volume_templates_called_once(self, mock_get):
 
     @mock.patch.object(ResourceClient, 'get')
     def test_get_reachable_volume_templates_called_once(self, mock_get):
-        uri = '/rest/storage-volume-templates/reachable-volume-templates?networks=\'/rest/fake\'&start=0&count=-1'
+        uri = '/rest/storage-volume-templates/reachable-volume-templates?networks=\'/rest/fake\'&start=0&count=-1&privateAllowedOnly=False'
```

Comment:
Do we need to add scopeUris test as well? as we did for other resources.

Reply:
No, its just a get end point.